### ISSUE TITLE
Is there a way to assign a page variable to one index.html file and use it?

### ISSUE THREAD

**Issue by JangCool** [IndexXuan/vite-plugin-mpa#18]
Is there a way to assign a page variable to one index.html file and use it?

I want to process by replacing only entry.js``` <script type="module" src="main.js">``` without changing the content of index.html. Is there any way to use the variable?
![image](https://user-images.githubusercontent.com/2518021/138227550-b2089ac9-5351-463e-ad9d-69b282d55b61.png)

In vue.config.js, it is used as follows:

![image](https://user-images.githubusercontent.com/2518021/138227755-50f87c67-736c-4606-bbb7-78501c706417.png)

**Comment by IndexXuan** [comment]
yes, use `vite-plugin-html-template`, the  index.html as a template should put into public folder.

I dont know the errors. maybe you can see the examples and find the different.
https://github.com/IndexXuan/vite-plugin-html-template/tree/main/examples/vite-mpa-project-non-standards

**Comment by JangCool** [comment]
@IndexXuan There is no difference from the link setting method you answered. By comparison, I am the js config and you are the typescript config.

um... Is there no way? I didn't know it would get blocked in the [yarn build]...  ;(

**Comment by JangCool** [comment]
@IndexXuan Don't you have the same error?  I'm wondering that about the processing situation. If not, I'll try use the index.html file individually.
